docs/refs/api/platform/callbacks_api.md

```diff
@@ -0,0 +1,19 @@
+### Callbacks
+
+A callback is a user provided function that a user may pass to an API. The callback allows the API to execute the user’s code in its own context.
+
+This is the technical reference for callbacks. You should read the [introduction to callbacks](callbacks.md) first to understand how to use them.
+
+### API
+
+[![View code](https://www.mbed.com/embed/?type=library)](https://docs.mbed.com/docs/mbed-os-api/en/mbed-os-5.5/api/classmbed_1_1Callback_3_01R_07A0_08_4.html)
+
+ ###### Thread example with callbacks
+
+The Callback API provides a convenient way to pass arguments to spawned threads.
+
+ [![View code](https://www.mbed.com/embed/?url=https://developer.mbed.org/teams/mbed_example/code/rtos_threading_with_callback/)](https://developer.mbed.org/teams/mbed_example/code/rtos_threading_with_callback/file/d4b2a035ffe3/main.cpp)
+
+###### Serial passthrough example with callbacks
+[![View code](https://www.mbed.com/embed/?url=https://developer.mbed.org/teams/mbed_example/code/rtos_threading_with_callback/)](https://developer.mbed.org/users/mbedAustin/code/SerialPassthrough/file/96cb82af9996/main.cpp)
+
```

Comment:
Should we put the sonar example here too now since it's hosted on the developer site?